src/fiatExchanges/ExternalExchanges.test.tsx

```diff
@@ -23,58 +21,18 @@ const mockStore = createMockStore({
   },
 })
 
-const MOCK_EXCHANGES_FETCH_RESPONSE: ExternalExchangeProvider[] = [
-  {
-    name: 'Bittrex',
-    link: 'https://bittrex.com/Market/Index?MarketName=USD-CELO',
-    currencies: [Currency.Celo, Currency.Dollar],
-    supportedRegions: ['global'],
-  },
-  {
-    name: 'CoinList Pro',
-    link: 'https://coinlist.co/asset/celo',
-    currencies: [Currency.Celo, Currency.Dollar],
-    supportedRegions: ['global'],
-  },
-  {
-    name: 'OKCoin',
-    link: 'https://www.okcoin.com/en/spot/trade/cusd-usd/',
-    currencies: [Currency.Celo, Currency.Dollar],
-    supportedRegions: ['global'],
-  },
-]
-
+// Removed mockFetch testing as fetchExchanges is called in SelectProvider
 describe('ExternalExchanges', () => {
-  const mockFetch = fetch as FetchMock
   beforeEach(() => {
     jest.clearAllMocks()
-    mockFetch.resetMocks()
   })
 
-  it('shows the correct UI when no provider is available', async () => {
-    mockFetch.mockResponse(JSON.stringify([]), { status: 200 })
-    const mockScreenProps = getMockStackScreenProps(Screens.ExternalExchanges, {
-      isCashIn: true,
-      currency: Currency.Celo,
-    })
-
-    const tree = render(
-      <Provider store={mockStore}>
-        <ExternalExchanges {...mockScreenProps} />
-      </Provider>
-    )
-
-    await flushMicrotasksQueue()
-
-    expect(tree.getByTestId('NoExchanges')).toBeTruthy()
-    expect(tree.getByTestId('SwitchCurrency')).toBeTruthy()
-    expect(tree.getByTestId('ContactSupport')).toBeTruthy()
-  })
+  // ExternalExchanges Screen will always have providers given restructuring of flow
```

Comment:
When leaving comments, its best to just talk about the current state of the code unless its crucial for understanding why things were done the way they were. For example, if it had to be done a certain way for backwards compatibility. 
```suggestion
  // ExternalExchanges Screen will always have providers passed in via props
```